RedfishPkg/RedfishCredentialDxe/RedfishCredentialDxe.c

```diff
@@ -93,7 +102,7 @@ RedfishCredentialGetAuthInfo (
 }
 
 /**
-  Notify the Redfish service provide to stop provide configuration service to this platform.
+  Notify the Redfish service provider tor stop provide configuration service to this platform.

```

Comment:
@igorkulchytskyy, wrong spell "tor"-> "to". :-)

Reply:
oops